fe/fe-core/src/main/java/com/starrocks/connector/partitiontraits/HivePartitionTraits.java

```diff
@@ -41,6 +41,11 @@ public boolean isSupportPCTRefresh() {
         return true;
     }
 
+    @Override
+    public String getTableName() {
+        return ((HiveMetaStoreTable) table).getTableName();
```

Comment:
Could you elaborate a little bit what's the difference compare to its original name

Reply:
origin name for hive resouce table is the name created at starrocks, it's not the name in hms